blt/blt.yml

```diff
@@ -99,6 +99,13 @@ command-hooks:
 simplesamlphp: true
 cgov:
   drupal_users_file: '${repo.root}/cgov-drupal-users.yml'
+
+validate:
+  deprecation:
+    modules: true
+    themes: true
+    profiles: true
```

Comment:
I don't know exactly. The default setting is:
```
  # You can change this to true to have blt automatically validate acsf-init.
  acsf: false
```

So this change I am making does not change the current settings one way or the other. It sounds like this is for a separate ticket. Also, it would be good to figure out what it actually does... if it is validating that the `acsf` configs and scripts are valid, then maybe it is good to have.

Reply:
Yeah, I was sorta of reading  that as more tightly related to this, but you're right, the `acsf` key is at the same level as `deprecation`, and not a subkey.  The new ticket is #3143.

I'll go ahead and mark this as approved, but will note there's some likelihood of future breakage. Depending on which of the 11.x versions you look at, there is some variation as to which of those tests are run.  For 11.4.2 (what we currently have) there are checks for `acsf`, `deprecation.modules`, and `deprecation.themes`.  In 11.x, there are no checks for the  `deprecation` key and in `main`, the entirety of `ValidateAllCommand` goes away.